runtime/src/main/java/org/corfudb/runtime/collections/Streamer.java

```diff
@@ -0,0 +1,42 @@
+package org.corfudb.runtime.collections;
+
+/**
+ * This is the callback interface that any client subscribe()ing to CorfuStore updates must implement.
+ *
+ * Created by hisundar on 2019-10-18
+ */
+public interface Streamer {
```

Comment:
nit: StreamListener might be a better name than Streamer

Reply:
done.